.github/actions/mirror/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'mirror'
```

Comment:
Let's give it a clear name. Also, the action dir should be renamed.

Reply:
The new name for the github action is 
"AWS Marketplace | ECR Image Mirror"

and the directory is now "aws_marketplace_ecr_mirror"